h2o-genmodel/src/main/java/hex/genmodel/AbstractMojoWriter.java

```diff
@@ -233,16 +235,21 @@ private void writeModelInfo() throws IOException {
     finishWritingTextFile();
   }
 
+  private static final Pattern QUOTE_PATTERN = Pattern.compile("[\"]{1}"); // Escape quotes
```

Comment:
{1} is a no-op a well as [] with just one member, it seems like only replaceAll("\"", "\"\"") would be enough.

Reply:
String.replaceAll does Pattern.compile all the time, so I'll avoid that. But yes, it can be simplified.